### PR TITLE
tiup: Update tiup-overview.md

### PR DESCRIPTION
**Pull request by shawn0915**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix: #14723  
Update tiup-overview.md, with the latest tiup version.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by kaaaaaaang — APPROVED** [review]

File: tiup/tiup-overview.md [hunk 1]
```diff
@@ -109,11 +117,17 @@ Use "tiup [command] --help" for more information about a command.
     - status：查看组件运行记录
     - clean：清除组件运行记录
     - mirror：从官方镜像克隆一个私有镜像
+    - telemetry：控制有关遥测的事情
```
Comment (qiancai) [inline tiup/tiup-overview.md:120]:
```suggestion
    - telemetry：控制遥测功能
```

File: tiup/tiup-overview.md [hunk 2]
```diff
@@ -109,11 +117,17 @@ Use "tiup [command] --help" for more information about a command.
...
+    - env：显示与 TiUP 相关的系统环境变量列表
+    - history：显示 TiUP 的历史执行记录，默认显示 100 行
+    - link：将组件二进制文件链接到 `$TIUP_HOME/bin/`
+    - unlink：取消组件二进制文件到 `$TIUP_HOME/bin/` 的链接
     - help：输出帮助信息
+    - completion：为指定的 shell (bash / zsh / fish / powershell) 生成自动补全脚本
```
Comment (qiancai) [inline tiup/tiup-overview.md:126]: 根据 https://docs.pingcap.com/zh/tidb/dev/tiup-command-completion， `completion` 目前只支持 bash 和 zsh 两种 shell 的命令补全， @kaaaaaaang PTAL

File: tiup/tiup-overview.md [hunk 3]
```diff
@@ -109,11 +117,17 @@ Use "tiup [command] --help" for more information about a command.
...
 - 可用的组件
-    - playground：在本机启动集群
-    - client：连接本机的集群
-    - cluster：部署用于生产环境的集群
+    - playground：在本机启动一个 TiDB 集群
+    - client：连接 playground 的客户端
```
Comment (qiancai) [inline tiup/tiup-overview.md:129]:
```suggestion
    - client：连接 TiUP Playground 的客户端
```

File: tiup/tiup-overview.md (hunk 2 above)
Comment (shawn0915) [reply tiup/tiup-overview.md:126]: 看来这个文档也需要更新了，tiup 目前引用的是 github.com/spf13/cobra v1.6.1 ，cobra 从 1.2.0 开始支持 fish, and PowerShell https://github.com/spf13/cobra/releases/tag/v1.2.0
ticloud cli 的文档已经更新了 https://github.com/pingcap/docs/blob/master/tidb-cloud/cli-reference.md?plain=1#L24

File: tiup/tiup-overview.md [hunk 4]
```diff
@@ -109,11 +117,17 @@ Use "tiup [command] --help" for more information about a command.
     - status：查看组件运行记录
     - clean：清除组件运行记录
     - mirror：从官方镜像克隆一个私有镜像
+    - telemetry：控制遥测功能
+    - env：显示与 TiUP 相关的系统环境变量列表
+    - history：显示 TiUP 的历史执行记录，默认显示 100 行
+    - link：将组件二进制文件链接到 `$TIUP_HOME/bin/`
+    - unlink：取消组件二进制文件到 `$TIUP_HOME/bin/` 的链接
     - help：输出帮助信息
+    - completion：为指定的 shell (bash / zsh / fish / powershell) 生成自动补全脚本
```
Comment (qiancai) [inline tiup/tiup-overview.md:126]:
```suggestion
    - completion：为指定的 shell (bash、zsh、fish、powershell) 生成自动补全脚本
```

File: tiup/tiup-overview.md [hunk 5]
```diff
@@ -109,11 +117,17 @@ Use "tiup [command] --help" for more information about a command.
     - status：查看组件运行记录
     - clean：清除组件运行记录
     - mirror：从官方镜像克隆一个私有镜像
+    - telemetry：控制遥测功能
+    - env：显示与 TiUP 相关的系统环境变量列表
+    - history：显示 TiUP 的历史执行记录，默认显示 100 行
+    - link：将组件二进制文件链接到 `$TIUP_HOME/bin/`
+    - unlink：取消组件二进制文件到 `$TIUP_HOME/bin/` 的链接
     - help：输出帮助信息
+    - completion：为指定的 shell (bash、zsh、fish、powershell) 生成自动补全脚本
```
Comment (qiancai) [inline tiup/tiup-overview.md:126]:
```suggestion
    - completion：为指定的 shell（bash、zsh、fish、powershell）生成自动补全脚本
```

**Review by qiancai — APPROVED** [review]

**Review by kaaaaaaang — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**